### ISSUE TITLE
Http workers stay in "invalid" status after ttl is reached

### ISSUE THREAD

**Issue by rustatian** [roadrunner-server/roadrunner#1669]
### Discussed in https://github.com/orgs/roadrunner-server/discussions/1668

<div type='discussions-op-text'>

<sup>Originally posted by **Aleksa1996** August  4, 2023</sup>
Hello,

I am having issues with ```http.pool.supervisor.ttl``` option after upgrade from ```v2023.1.2``` to ```v2023.2.1```
When workers reach ttl, they stay in status "invalid". I am expecting restart and them to become in status ready again. But they are stuck in that invalid status.

Output from command ```bin/rr workers -i```

![image](https://github.com/roadrunner-server/roadrunner/assets/26756429/5e199dff-6aaf-4b92-85ed-6af435483310)

This is the config that i am using:

```yaml
version: '3'

server:
    command: "php public/index.php"
    env:
        - APP_RUNTIME: Baldinof\RoadRunnerBundle\Runtime\Runtime
        - APP_ENV: prod

rpc:
    listen: tcp://127.0.0.1:6001

http:
    address: 0.0.0.0:80
    middleware: [ "static", "gzip" ]
    uploads:
        forbid: [ ".php", ".exe", ".bat", ".sh" ]
    static:
        dir: "public"
        forbid: [ ".php", ".htaccess" ]
    pool:
        num_workers: 12
        debug: false
        max_jobs: 100
        supervisor:
            watch_tick: 1s
            ttl: 1800s
            idle_ttl: 0s
            max_worker_memory: 350
            exec_ttl: 180s

status:
    address: localhost:2114

logs:
    mode: production
    output: stderr

metrics:
    address: "0.0.0.0:9180"
```

Thanks for this wonderful project.</div>

**Comment by rustatian** [comment]
@Aleksa1996 I've found the problem, will be fixed and released in `v2023.2.2` next Thursday.

**Comment by rustatian** [comment]
Fixed by: https://github.com/roadrunner-server/sdk/pull/84
